### ISSUE TITLE
When using screen reader, navigation buttons not announcing month/year when CaptionLayout is dropdown

### ISSUE THREAD

**Issue by erstrauch** [gpbl/react-day-picker#2715]
Thanks for reporting a bug for DayPicker. Please provide here a brief description of the issue you’ve encountered. 

## To reproduce

```tsx
import { DayPicker } from "react-day-picker";
import "react-day-picker/style.css";

export default function App() {
  return <DayPicker mode="single"  captionLayout="dropdown" />;
}
```

## Actual Behavior

When utilizing a screen reader, using the navigation buttons doesn't announce the month/year when the captionLayout is dropdown

## Expected Behavior

Using the navigation buttons should announce the month/year when the captionLayout is dropdown, just like it does when the captionLayout is label
